hooli_data_eng/resources/__init__.py

```diff
@@ -64,6 +64,12 @@ def get_env():
 # Similar to having different dbt targets, here we create the resource
 # configuration by environment
 
+dbt_resource = DbtCliResource(
```

Comment:
This is a little confusing to me, this resource is only used in the local case right? And only for manifest generation? Whereas the other resources here, including the other use of dbt, are all specific to local, branch, or prod. 

I'd maybe consider moving this to within the if block you added

Reply:
great callout, wasn't satisfied with this overall (I was following the examples in Dagster & dbt for consistency, but what I actually wanted was to use the `DbtArtifact` class I mentioned in the comments. With an assist in Slack I was able to get that up and running 🎉 ,so this code is gone now